rclcpp/include/rclcpp/experimental/intra_process_manager.hpp

```diff
@@ -207,7 +207,7 @@ class IntraProcessManager
       sub_ids.take_shared_subscriptions.size() <= 1)
     {
       // There is at maximum 1 buffer that does not require ownership.
-      // So we this case is equivalent to all the buffers requiring ownership
+      // So this case is equivalent to all the buffers requiring ownership
```

Comment:
nit: this change is unrelated to the fix that this PR is proposing. I'd rather see this change reverted for this PR. This keeps the PR focused and reduces overhead/confusion when generating changlogs (e.g. it will cause this PR to appear in both `rclcpp`'s and `rclcpp_action`'s changelogs)